Clinical-Genomics/scout
# Direct links to chanjo2 external app

This PR adds a functionality or fixes a bug.
- Fix #4539

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>


**How to test**:
1. Deploy on stage together with [this branch of chanjo2](https://github.com/Clinical-Genomics/chanjo2/pulls)
2. Use this case [safeguinea](https://scout-stage.scilifelab.se/cust000/F0006358-mip7)
3. Check the following chanjo2 reports:
- The one created for the case default panels, link on case sidebar:
<img width="200" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/52b4a06b-b541-4f4e-973d-8ec263baffc3">

- The ones present on case page when you create an HPO gene panel:
<img width="500" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/3dd295b8-d55a-495f-9e5e-a899cfbe5699">

- The ones present on gene panel page, after you click on a gene panel present on case page:

<img width="500" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/76a0228a-25c3-4207-a58b-3d1d8e437638">
<img width="500" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/a9b21a51-3c40-40c3-abfb-e3b02fcb59ba">



**Expected outcome**:
The links should be on pages external to scout, so that links present on these pages will work as expected

**Review:**
- [x] tests executed by CR, Jakob37
